packages/server/README.md

```diff
@@ -61,3 +64,10 @@ You can also run in `watch` mode
 ## runs and watches only this one test file
 npm run test-watch ./test/unit/api_spec.coffee
 ```
+
+To run an individual e2e test:
+
+```bash
+## runs only this one test file
+npm run test-e2e -- --spec ./test/e2e/base_url_spec.coffee
```

Comment:
Can be shorted to just `npm run test-e2e -- --spec base_url` since it will use String::includes